packages/neptune-extension/src/common/components/BootstrapApp.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
+import {Store} from 'redux';
 import {Provider} from 'react-redux';
 
 import configureStore from 'common/state/store';
 import App, {AppProps} from './App';
 
-const store = configureStore();
-
 export const BootstrapApp: React.FC<AppProps> = ({
   platformNotebook
-}) => (
-  <Provider store={store}>
-    <App platformNotebook={platformNotebook} />
-  </Provider>
-);
+}) => {
+  const storeRef = React.useRef<Store>();
```

Comment:
Type `Store` is not proper here as this is generic.

If you will type:
```
else {
    const state = storeRef.current.getState();
  }
```

Then `state` will be `any`.
So whole state object now is passed to the app as any.

Previously it worked because TS inferred type from `configureStore` return type.

Lets do it like this:

`const storeRef = React.useRef<ReturnType<typeof configureStore>>();`

In this case when I use it:
`const state = storeRef.current.getState();`

then `state` will have type compatible with `AppState`.
This means TS will understand all the properties returned in each reducer.